pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -540,12 +540,16 @@ const inputSelectorToPipelineTemplate = `{{- define "inputSelectorToPipelineTemp
 <label {{sourceTypelabelName .Source}}>
   <match **>
     @type label_router
-{{- if .PipelineNames }}
-    default_route {{sourceTypelabelName .Source}}_DEFAULT
-{{- end }}
     {{- range .InputSelectors }}
     {{ . }}
     {{- end}}
+{{- if .PipelineNames }}
+    <route>
+      @label {{sourceTypelabelName .Source}}_DEFAULT
```

Comment:
What is sourcetype_default?  "Default" is whatever the default output type is, currently elasticsearch.  I believe the implication of what you are saying is:  If I have no other routing choices defined but I need to send everything else to the default store do that here, in which case it should be "@DEFAULT"

Reply:
> What is sourcetype_default? "Default" is whatever the default output type is, currently elasticsearch.

"_default" is my wrong choice. As you said, "If I have no other routing choices defined but I need to send everything else"
is what I want to say. But it's not limited to "the default store". I think it's same as the built-in input name  `application`.
So is the "_BUILT_IN" the right choice?

Or are there any good words?
To implement pod label filtering, the commit 41bb627a2b332e6297f52920f85c0f94bbc04071 is replaced with the commit 5b7c7f599651b91017412be99532444c51023dae.
I want to define the following "<match **> section in the commit 41bb627a2b332e6297f52920f85c0f94bbc04071.

Quoted from the commit 41bb627a2b332e6297f52920f85c0f94bbc04071:
```
  # Relabel specific sources (e.g. logs.apps) to multiple pipelines
  <label @_APPLICATION>
     <match kubernetes.**_dev-apple_**>
      @type copy
      <store>
            @type relabel
            @label @APPS_PIPELINE2
      </store>
    </match>
    <match kubernetes.**_project1-namespace_**>
      @type copy
      <store>
                @type relabel
                @label @APPS_PIPELINE
      </store>
    </match>
    <match kubernetes.**_project2-namespace_**>
      @type copy
      <store>
                @type relabel
                @label @APPS_PIPELINE
      </store>
      <store>
                @type relabel
                @label @APPS_PIPELINE2
      </store>
    </match>
+   <match **>
|     @type copy
|     <store>
|           @type relabel
|           @label @MY_DEFAULT_PIPELINE
|     </store>
+   </match>                    
   </label>                         
```

Reply:
I think the code is correct but agree the name is confusing.

The bug we are fixing is that the label filter `default_route` only gets _unmatched_ traffic, but we want inputs with no filters to forward _all_ traffic, so @k-keiichi-rh  has replaced `default_route` with a "match all" route. 
How about `{{sourceTypelabelName .Source}}_ALL`, so we have `@_APPLICATION_ALL` meaning the label that gets all the application logs with no filters.

Reply:
Thanks. I fixed to `{{sourceTypelabelName .Source}}_ALL`.